contracts/mainnet/connectors/uniswapV3/main.sol

```diff
@@ -0,0 +1,203 @@
+pragma solidity ^0.7.6;
+pragma abicoder v2;
+
+/**
+ * @title Uniswap v3.
+ * @dev Decentralized Exchange.
+ */
+
+import {TokenInterface} from "../../common/interfaces.sol";
+import {Helpers} from "./helpers.sol";
+import {Events} from "./events.sol";
+
+abstract contract UniswapResolver is Helpers, Events {
+    /**
+     * @dev Mint New Position
```

Comment:
@cryptoDev222 Can you add comments for the below params? I have converted struct to normal params